lib/premailex.ex

```diff
@@ -14,9 +14,9 @@ defmodule Premailex do
       "<html><head><style>p{background-color: #fff;}</style></head><body><p style=\\\"background-color:#fff;color:#000;\\\">Text</p></body></html>"
 
   """
-  @spec to_inline_css(String.t()) :: String.t()
-  def to_inline_css(html) do
-    Premailex.HTMLInlineStyles.process(html)
+  @spec to_inline_css(String.t(), String.t()) :: String.t()
+  def to_inline_css(html, css_selector \\ "style,link[rel=\"stylesheet\"][href]") do
```

Comment:
I would make the second argument an optional Keyword list:

```elixir
@spec to_inline_css(String.t(), Keyword.t()) :: String.t()
def to_inline_css(html, options \\ []) do
```

And set the default in the `HTMLInlineStyles`:

```elixir
@spec process(String.t(), Keyword.t()) :: String.t()
def process(html, options \\ []) do
  css_selector = Keyword.get(options, :css_selector, "style,link[rel=\"stylesheet\"][href]")
```